### ISSUE TITLE
Docker image tags based on kafka version

### ISSUE THREAD

**Issue by rmelick** [spotify/docker-kafka#61]
It would be awesome if the docker image had more tags than just "latest".  How difficult would it be for there to be different tags for different versions?  I'm working with some other systems to do not yet support kafka 0.10.

**Comment by bluepuma77** [comment]
I agree, it would be great to have an old image with kafka 0.8 still available on docker hub.

**Comment by catkins** [comment]
Would also love this, `0.9` specifically

**Comment by NeQuissimus** [comment]
This is indeed keeping us from using the Spotify image right now, as we need reproducible builds.
